tools/bin/load_test/load_test_airbyte.sh

```diff
@@ -0,0 +1,184 @@
+
+  load_test_airbyte performs a load-test against an existing Airbyte instance.
+
+  Available OPTIONs:
+
+  ${CLEAR}-h
+    ${GREEN}Display help
+
+  ${CLEAR}-W <workspace id>
+    ${GREEN}Specify the workspace ID where new connectors and connections should be created.
+    Required.
+
+  ${CLEAR}-H <hostname>
+    ${GREEN}Specify the Airbyte API server hostname that the script should call to create new connectors and connections.
+    Defaults to 'localhost'.
+
+  ${CLEAR}-P <port>
+    ${GREEN}Specify the port for the Airbyte server. If kube (ie. '-k' is provided), port-forwarding
+    will be set up for the airbyte-server deployment using the provided <port> (ie. '8001:8001').
+    Defaults to '8001'.
+
+  ${CLEAR}-C <count>
+    ${GREEN}Specify the number of connections that should be created by the script.
+    Defaults to '1'.
+
+  ${CLEAR}-T <minutes>
+    ${GREEN}Specify the time in minutes that each connection should sync for.
+    Defaults to '10'.
+
+  ${CLEAR}-k
+    ${GREEN}Indicate that the script is running against a kubernetes instance of Airbyte
+
+  ${CLEAR}-N <namespace>
+    ${GREEN}Specify the kubernetes namespace where the airbyte-server deployment exists g (ex. "ab").
+    Only use with '-k' option.
+    Defaults to 'default'.
+  """ && exit 1
+}
+
+if [[ $# -eq 0 ]] ; then
+    showhelp
+    exit 0
+fi
+
+hostname=localhost
+api_port=8001
+num_connections=1
+sync_minutes=10
+kube=false
+kube_namespace=default
+
+while getopts "hW:H:P:C:T:kN:" options ; do
+  case "${options}" in
+    h)
+      showhelp
+      ;;
+    W)
+      workspace_id="${OPTARG}"
+      ;;
+    H)
+      hostname="${OPTARG}"
+      ;;
+    P)
+      api_port="${OPTARG}"
+      ;;
+    C)
+      num_connections="${OPTARG}"
+      ;;
+    T)
+      sync_minutes="${OPTARG}"
+      ;;
+    k)
+      kube=true
+      ;;
+    N)
+      if test "$kube" = true; then
+        kube_namespace="${OPTARG}"
+      else
+        echo "error: -k must be set to use option -N"
+        exit 1
+      fi
+      ;;
+    *)
+      showhelp
+      ;;
+  esac
+done
+
+if test -z "$workspace_id"; then
+  echo "error: must set a workspace id with -W"
+  exit 1
+fi
+
+echo "set workspace_id to ${workspace_id}"
+echo "set hostname to ${hostname}"
+echo "set api_port to ${api_port}"
+echo "set num_connections to ${num_connections}"
+echo "set sync_minutes to ${sync_minutes}"
+echo "set kube to ${kube}"
+echo "set kube_namespace to ${kube_namespace}"
+
+# base64-encoded: {"user_id": "cloud-api", "email_verified": "true"}
+api_header="eyJ1c2VyX2lkIjogImNsb3VkLWFwaSIsICJlbWFpbF92ZXJpZmllZCI6ICJ0cnVlIn0K"
+
+# call the API with the endpoint passed as arg $1, and (optional) payload passed as arg $2
+# example of calling the API with a payload:
+#    callApi "destinations/list" "{\"workspaceId\":\"${workspace}\"}"
+function callApi {
+  curl -s -X POST -H 'Content-Type: application/json' -H "X-Endpoint-API-UserInfo: ${api_header}" -d "$2" "${hostname}:${api_port}/api/v1/$1"
```

Comment:
note: `--silent` suppresses errors which can be sketchy for debugging.  you can add `--show-error` to enable showing errors with silent enabled

two style notes here, long form flags are meant for scripts to save people from the man pages where possible, and very long lines I think are easier to reason about on multiple lines.
```suggestion
  endpoint=$1
  payload=$2
  curl 
    --silent \ 
    --request POST  \
    --header 'Content-Type: application/json'  \
    --header "X-Endpoint-API-UserInfo: ${api_header}" \
    --data "${payload}" \
    "${hostname}:${api_port}/api/v1/${endpoint}"
```